server/chat/backend/agent/utils/persistence/context_manager.py

```
@@ -205,7 +206,7 @@ async def flush_session(cls, session_id: str) -> bool:
         return True
 
     @classmethod
-    def cleanup(cls):
-        """Cleanup resources on shutdown."""
+    async def cleanup(cls):
```

Ruff (0.15.10)

[warning] Missing return type annotation for classmethod `cleanup`

Add return type annotation: `None`

(ANN206)